api/envoy/config/trace/v2/trace.proto

```diff
@@ -80,6 +80,17 @@ message DynamicOtConfig {
   google.protobuf.Struct config = 2;
 }
 
+// Configuration for the Datadog tracer.
```

Comment:
Can we add a release note for this? Are there any other docs that need updating? Perhaps https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/tracing?